### ISSUE TITLE
GetOutputDevice() always returns 0 in vulkan

### ISSUE THREAD

**Issue by timon37** [ValveSoftware/openvr#604]
We've got a system with both amd and nvidia gpus in it, and GetOutputDevice() always returns 0 (we've tried calling it at a few different times).
Lib version is 1.0.9, of course we're calling the function with vr::TextureType_Vulkan, and everything in the system we could manage to update was up to date.

Are there any requirements for when/how the function should be called? Or a sample using it (a.k.a. did anyone ever test it works)?

**Comment by danginsburg** [comment]
@timon37 Yes, sorry to say this is not working in the current release of SteamVR, but I have fixed it.  It also is going to require a new OpenVR SDK release because GetOuptutDevice needs another parameter (VkInstance).  

If you are really desperate to use it before the next release, you could use TextureType_DirectX on Windows to get the LUID then use VK_KHR_get_physical_device_properties2/VK_KHR_external_memory_capabilities to get the matching VkPhysicalDeviceIDPropertiesKHR.deviceLUID.  However, if you can wait, the fix will just give you back the VkPhysicalDevice, and it will work on Windows or Linux requiring no platform-specific code.

Sorry for the troubles.

**Comment by highyun** [comment]
Hi danginsburg,is the issue still have not fixed?
